core/ipc/sink/common/src/main/java/org/opennms/core/ipc/sink/common/SinkStrategy.java

```diff
@@ -36,9 +36,12 @@
 
     private static final String KAFKA_SINK_STRATEGY_NAME = "kafka";
 
+    private static final String AWS_SINK_STRATEGY_NAME = "aws";
+
     public static enum Strategy {
         CAMEL(CAMEL_SINK_STRATEGY_NAME, "JMS implementation using Camel"),
-        KAFKA(KAFKA_SINK_STRATEGY_NAME, "Kafka implementation using the Kafka consumer/producer APIs");
+        KAFKA(KAFKA_SINK_STRATEGY_NAME, "Kafka implementation using the Kafka consumer/producer APIs"),
+        AWS(AWS_SINK_STRATEGY_NAME, "AWS SQS implementation using the AWS JMS consumer/producer APIs");
```

Comment:
Can we rename this enum and the related constants to `SQS` rather than `AWS`? We can also remove the reference to JMS from the description.

Reply:
No problem! should I also rename the actual value (currently, "aws") ? I'm asking as that also means I'd have to tweak the documentation.

Reply:
Speaking about docs changes, what about `AmazonSQSSinkConstants.AWS_CONFIG_PID` ? currently it defaults to "org.opennms.core.ipc.sink.aws". Should I rename it to "org.opennms.core.ipc.sink.sqs" ?

Reply:
Yeah, I'd rename the "aws" value too.

As for the pid, I'd opt for `org.opennms.core.ipc.sink.sqs` or `org.opennms.core.ipc.sink.awq.sqs`.

Reply:
ok!

Reply:
Final question, the feature itself to be installed on Minion is called `opennms-core-ipc-sink-aws`, should I also rename that to be `opennms-core-ipc-sink-sqs` ?

Reply:
Disregard the last part, it is called `opennms-core-ipc-sink-aws-sqs`, so we're good.